src/Cello/monitor_Monitor.cpp

```diff
@@ -9,7 +9,8 @@
 
 #include "monitor.hpp"
 
-#include "auto_config.def"
+// TODO: adapt to use cmake generated information
+//#include "auto_config.def"
```

Comment:
How important is this? Is it best to leave this to be addressed later?

Reply:
I'll address this today.

Reply:
I now reintroduced the information that was (almost) straightforwardly available.
It's still not 100% but I think it's a good starting point and if people thing important items are missing we can reintroduce those later.